### PR TITLE
Code Consistency: luminosity --> light_range

### PR DESCRIPTION
**Pull request by Firecage**
Well, the title basically says what this PR does. Change the remaining luminosity vars in applicable code to light_range.

### CODE REVIEW

**Comment by ma44** [comment]
>branch name

File: code/game/machinery/lightswitch.dm [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	var/on = TRUE
 	var/area/area = null
 	var/otherarea = null
-	//	luminosity = 1
+	//	light_range = 1
```
Comment (Cyberboss) [inline code/game/machinery/lightswitch.dm]: just remove the line

File: code/modules/power/apc.dm [hunk 2]
```diff
@@ -78,7 +78,7 @@
 	powernet = 0		// set so that APCs aren't found as powernet nodes //Hackish, Horrible, was like this before I changed it :(
 	var/malfhack = 0 //New var for my changes to AI malf. --NeoFite
 	var/mob/living/silicon/ai/malfai = null //See above --NeoFite
-//	luminosity = 1
+//	light_range = 1
```
Comment (Cyberboss) [inline code/modules/power/apc.dm]: ^

File: code/modules/power/singularity/singularity.dm [hunk 3]
```diff
@@ -278,7 +278,7 @@
 		desc = "[initial(desc)] It glows fiercely with inner fire."
 		name = "supermatter-charged [initial(name)]"
 		consumedSupermatter = 1
-		luminosity = 10
+		light_range = 10
```
Comment (Cyberboss) [inline code/modules/power/singularity/singularity.dm]: You need to use set_light here instead

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Firecage** [comment]
Sure, will make the changes

**Comment by optimumtact** [comment]
no RUSH

**Comment by Firecage** [comment]
Oranges. Can you perhaps unclose this? Will finish it up tomorrow. I actually forgot about this PR.

**Comment by Firecage** [comment]
Ahwell then.

**Comment by AnturK** [comment]
@Firecage please

**Comment by Firecage** [comment]
I...didn't even see anyone reopen this to be honest. Since after oranges ignored me....I gave up and didn't get any notifications...

**Comment by optimumtact** [comment]
I didn't get your notification or ignored it, did you try to reach me in IRC I can be much more easily reached with a pm there.

**Comment by Firecage** [comment]
Alright @optimumtact , PR is finally doneso.

File: code/modules/power/singularity/singularity.dm [hunk 4]
```diff
@@ -278,7 +278,7 @@
 		desc = "[initial(desc)] It glows fiercely with inner fire."
 		name = "supermatter-charged [initial(name)]"
 		consumedSupermatter = 1
-		luminosity = 10
+		set_light = 10
```
Comment (ShizCalev) [inline code/modules/power/singularity/singularity.dm]: set_light(10)

**Review by ShizCalev — DISMISSED** [review]

**Comment by Cyberboss** [comment]
@Firecage !!!

**Comment by optimumtact** [comment]
fool me once, shame on me

fool me twice, shame on me

**Comment by Firecage** [comment]
Well sorry guys. The past two weeks I have been dead tired by the time it gets night, which is what the time is here now. And otherwise in general been busy the entire day.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**